R/findLastRun.R

```diff
@@ -0,0 +1,31 @@
+#' Determine the latest path of a run
+#'
+#' Search the given output folder for the run with the most recent time stamp
+#' and return the path to this run.
+#' If the output folder contains only one run, return its path, also if it does not contain a time stamp.
+#'
+#' @author Ricarda Rosemann
+#'
+#' @param outputFolder character, output folder to search in
+#'
+findLastRun <- function(outputFolder) {
+
+  if (dir.exists(outputFolder)) {
+    paths <- list.dirs(outputFolder, recursive = FALSE)
+    regexStamp <- "\\d{4}-\\d{2}-\\d{2}_\\d{2}\\.\\d{2}\\.\\d{2}"
+    stamp <- sub(paste0("^.*_(", regexStamp, ")$"), "\\1", paths)
+    if (length(stamp) > 0) {
+      path <- paths[which(stamp == max(stamp))]
+    } else if (length(paths) == 1) {
+      path <- paths
```

Comment:
This case is covered by the above, right?
```suggestion
```

Reply:
You're right. But actually this also means that paths without a time stamp are included in the search for the most recent path, with unclear outcome. I therefore adapted the definition of ```stamp```, which makes this line necessary again.
So thanks a lot for this comment, as it helped to avoid a potentially annoying bug!